docs/src/content/docs/reference/Config File/packages.md

```diff
@@ -162,6 +162,12 @@ version: 1.0.0
 
 `dependency` isn't yet supported.
 
+### `tauri.conf.json`
+
+Supports [Tauri configuration files](https://v1.tauri.app/v1/references/configuration-files) in json format. Also supports `tauri.macos.conf.json`, `tauri.windows.conf.json`, and `tauri.linux.conf.json`.
+
+`dependency` isn't supported yet.
```

Comment:
```suggestion
```

I don't think there is anything dependency-like to version in here in the future.